wallet/wallet.c

```diff
@@ -1135,60 +1136,62 @@ static bool wallet_channels_load_active(struct wallet *w)
 
 	/* We load all channels */
 	stmt = db_prepare_v2(w->db, SQL("SELECT"
-					"  id"
-					", peer_id"
-					", short_channel_id"
-					", full_channel_id"
-					", channel_config_local"
-					", channel_config_remote"
-					", state"
-					", funder"
-					", channel_flags"
-					", minimum_depth"
-					", next_index_local"
-					", next_index_remote"
-					", next_htlc_id"
-					", funding_tx_id"
-					", funding_tx_outnum"
-					", funding_satoshi"
-					", our_funding_satoshi"
-					", funding_locked_remote"
-					", push_msatoshi"
-					", msatoshi_local"
-					", fundingkey_remote"
-					", revocation_basepoint_remote"
-					", payment_basepoint_remote"
-					", htlc_basepoint_remote"
+					"  id" // 0
+					", peer_id" // 1
+					", short_channel_id" // 2
+					", full_channel_id" // 3
+					", channel_config_local" // 4
+					", channel_config_remote" // 5
+					", state" // 6
+					", funder" // 7
+					", channel_flags" // 8
+					", minimum_depth" // 9
+					", next_index_local" // 10
+					", next_index_remote" // 11
+					", next_htlc_id" // 12
+					", funding_tx_id" // 13
+					", funding_tx_outnum" // 14
+					", funding_satoshi" // 15
+					", our_funding_satoshi" // 16
+					", funding_locked_remote" // 17
+					", tx_sigs_remote" // 18
```

Comment:
Please add new fields at the bottom of the statement, that avoids a whole cascade of index updates and minimizes the diff.